lib/defaults/config.json

```diff
@@ -84,5 +84,6 @@
   "alwaysHostPublic": false,
   "checkDependencies": true,
   "shutdownTimeout": 30000,
-  "cores": 1
+  "cores": 1,
+  "urlPrefix": false
```

Comment:
Shouldn't the default be `/`? Otherwise we have a mix of types between boolean and string

```suggestion
  "urlPrefix": "/"
```